### PR TITLE
Add presence for VUDU

### PR DESCRIPTION
**Pull request by SmartieCodes**
This presence is made for https://www.vudu.com/
It'll create a rich presence for whenever a user is watching a movie on VUDU.

### CODE REVIEW

**Comment by SmartieCodes** [comment]
> 
> 
> Resolve this:
> image

I have changed that, thank you for pointing that out!

File: VUDU/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      "id": "374245143655612428"
+    },
+    "url": "www.vudu.com",
+    "description": {
+      "en": "This extension adds rich presence support to VUDU, a movie site."
+    },
+    "service": "VUDU",
+    "version": "1.0.0",
+    "logo": "https://www.vudu.com/content/images/logo.svg",
+    "thumbnail": "https://cdn.discordapp.com/attachments/377284586326982666/697982989048741928/unknown.png",
```
Comment (Bas950) [inline VUDU/dist/metadata.json:13]: upload to imgur

File: VUDU/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "author": {
+      "name": "Autumn",
+      "id": "374245143655612428"
+    },
+    "url": "www.vudu.com",
+    "description": {
+      "en": "This extension adds rich presence support to VUDU, a movie site."
```
Comment (Bas950) [inline VUDU/dist/metadata.json:8]: should be infomation about what the site is

Comment (Bas950) [reply VUDU/dist/metadata.json:8]: could probs get that off google

File: VUDU/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "service": "VUDU",
+    "version": "1.0.0",
+    "logo": "https://www.vudu.com/content/images/logo.svg",
+    "thumbnail": "https://cdn.discordapp.com/attachments/377284586326982666/697982989048741928/unknown.png",
+    "color": "#236cba",
+    "tags": [
+      "movies"
+    ],
+    "category": "videos",
+    "iframe": false
```
Comment (Bas950) [inline VUDU/dist/metadata.json:19]: iframe false isnt needed since iframe is optional

Dont forget to remove the comma behind category when you remove it

**Comment by Bas950** [comment]
also upload proof of it working

**Comment by SmartieCodes** [comment]
> 
> 
> also upload proof of it working


Here you go, if you need more proof please feel free to tell me!

https://youtu.be/Nq2lq11KkLU

**Pull request merged**